lib/activerecord-import/import.rb

```diff
@@ -285,32 +285,7 @@ def supports_setting_primary_key_of_imported_objects?
     # below for what +options+ are available.
     #
     # == Options
-    # * +validate+ - true|false, tells import whether or not to use
```

Comment:
Thoughts on this decision here? Also, thoughts on removing more of the documentation here as it's examples and we could link to the examples in the README?

Reply:
I don't think we want to remove the options from the comments here because they get picked up in the ruby doc:

https://www.rubydoc.info/github/zdennis/activerecord-import/ActiveRecord%2FBase.import

Reply:
That makes a lot of sense 👍